tutorials/src/bin/basic-tutorial-7.rs

```diff
@@ -0,0 +1,66 @@
+extern crate gstreamer as gst;
+use gst::prelude::*;
+
+fn main() {
+    // Initialize GStreamer
+    if let Err(err) = gst::init() {
+        eprintln!("Failed to initialize Gst: {}", err);
+        return;
+    }
+
+    let audio_source = gst::ElementFactory::make("audiotestsrc", "audio_source").unwrap();
+    let tee = gst::ElementFactory::make("tee", "tee").unwrap();
+    let audio_queue = gst::ElementFactory::make("queue", "audio_queue").unwrap();
+    let audio_convert = gst::ElementFactory::make("audioconvert", "audio_convert").unwrap();
+    let audio_resample = gst::ElementFactory::make ("audioresample", "audio_resample").unwrap();
+    let audio_sink = gst::ElementFactory::make("autoaudiosink", "audio_sink").unwrap();
+    let video_queue = gst::ElementFactory::make("queue", "video_queue").unwrap();
+    let visual = gst::ElementFactory::make("wavescope", "visual").unwrap();
+    let video_convert = gst::ElementFactory::make("videoconvert", "video_convert").unwrap();
+    let video_sink = gst::ElementFactory::make ("autovideosink", "video_sink").unwrap();
+
+    let pipeline = gst::Pipeline::new("test-pipeline");
+
+    audio_source.set_property("freq", &215.0).unwrap();
+    visual.set_property_from_str("shader", "0");
+    visual.set_property_from_str("style", "1");
```

Comment:
I'd prefer using `set_property()` always. Similar to here https://github.com/sdroege/gstreamer-rs/blob/b773c0b46a75ecccc21841abf317d94ea2a116e4/examples/src/bin/playbin.rs#L24 you can also do that with glib::EnumClass

Reply:
Well, this is much more code for the same thing, I would rather adviertise using `set_property_from_str` tbh.

Reply:
Via GEnumClass you can get proper error handling, this here more or less silently fails. I'm fine with keeping it, but then please use the proper name ("none" and "lines") instead of the integer values :)

Reply:
Well yeah, but has you told me yesterday, this never fails unless you fail at writing :-)